backend/app/training/find_sitemap.py

```
@@ -2,12 +2,12 @@
 from bs4 import BeautifulSoup
 import re
 import itertools
-
+SITE_MAP_LITERAL = "sitemap_index.xml"
 # List of common sitemap names
 common_sitemaps = [
-    "sitemap.xml", "sitemap_index.xml", "sitemap.xml.gz", "sitemap_index.xml.gz",
+    "sitemap.xml", SITE_MAP_LITERAL, "sitemap.xml.gz", "sitemap_index.xml.gz",
     "sitemap.html", "sitemap.php", "feeds/posts/default?orderby=updated",
-    "sitemap_index.html", "sitemap_index", "sitemap-index.xml", "sitemap_index.xml"
+    "sitemap_index.html", "sitemap_index", "sitemap-index.xml", SITE_MAP_LITERAL
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)